src/pureObserver.spec.ts

```diff
@@ -5,142 +5,174 @@ import {pureObserver} from './useObserver'
 test('it reacts to a change', () => {
   const object = {test: 'abc'}
   const obj1Callback = td.func() as () => void
+
   pureObserver(object, obj1Callback)
 
+  expect(object.test).toEqual('abc')
+  td.verify(obj1Callback(), {times: 0})
+
   object.test = 'def'
-  td.verify(obj1Callback())
+  expect(object.test).toEqual('def')
+  td.verify(obj1Callback(), {times: 1})
+
   object.test = 'there'
+  expect(object.test).toEqual('there')
   td.verify(obj1Callback(), {times: 2})
 })
 
 test('that it can work with multiple objects, no react', () => {
   const obj1 = {
     foo: 'here',
+    // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
     mutateMe: () => {
       obj1.foo = 'there'
     },
   }
   const obj2 = {
     bar: 'pete',
+    // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
we could put it at the top of the file to make the linter less noise'y for tests

Reply:
it's only there twice. let's do it on the third repeat